### ISSUE TITLE
App incorrect work on iPhone with iOS 15.0+

### ISSUE THREAD

**Issue by Maschenko** [Scirra/Construct-3-bugs#5097]
## Problem description

Constructed games do not work correctly on iOS. Some of the animations do not work at all, the rest do not work correctly. The sounds built into the game disappear, the games periodically slow down. Testing took place on different iPhones from 6s to 13 Pro.

## Attach a .c3p

All games

## Steps to reproduce

1. Open any game created in the Сonstruct 3

## Observed result

The application does not work the same as before iOS 15, after iOS 15 the application does not work correctly.

## Expected result

Should work correctly.

## More details

Constructed games do not work correctly on iOS. Some of the animations do not work at all, the rest do not work correctly. The sounds built into the game disappear, the games periodically slow down. Testing took place on different iPhones from 6s to 13 Pro.

**Comment by fredriksthlm** [comment]
I understand this is kind of "weak" bug report.

Few comments to add to this from what I have noticed this week. I also see the same, after updating to **IOS 15.0.1**.
It worked fine in all ios versions up to 15.0, but after the 15.0.1 sounds don't work for me any longer.
Even games exported and released long time ago now stopped working. Also the loading time increased a lot.

I understand that this is an IOS bug/issue and not Construct, but maybe you can verify if you see the same, and can give some hints what has happened...

**Comment by Maschenko** [comment]
> Я так понимаю, это своего рода "слабый" отчет об ошибке.
> 
> Несколько комментариев, которые я заметил на этой неделе, чтобы добавить к этому. Я тоже вижу то же самое после обновления до **IOS 15.0.1** . Он отлично работал во всех версиях ios до 15.0, но после 15.0.1 звуки у меня перестали работать. Даже игры, давно экспортированные и выпущенные, перестали работать. Также значительно увеличилось время загрузки.
> 
> Я понимаю, что это ошибка / проблема IOS, а не Construct, но, возможно, вы можете проверить, видите ли вы то же самое, и можете дать некоторые подсказки, что произошло ...

**Comment by Maschenko** [comment]
> I understand this is kind of "weak" bug report.
> 
> Few comments to add to this from what I have noticed this week. I also see the same, after updating to **IOS 15.0.1**. It worked fine in all ios versions up to 15.0, but after the 15.0.1 sounds don't work for me any longer. Even games exported and released long time ago now stopped working. Also the loading time increased a lot.
> 
> I understand that this is an IOS bug/issue and not Construct, but maybe you can verify if you see the same, and can give some hints what has happened...

**Comment by AshleyScirra** [comment]
I'm afraid you'll have to be more specific in bug reports. Since the report does not follow the guidelines, I cannot test what you're seeing. So I just opened Space Blaster on an iPad with iOS 15 and everything seems to work fine. I also updated it to iOS 15.0.1 and tried again, and it still worked just fine.

This is why we have the bug report guidelines and why they are required: we need that information to be able to help. As there's not enough information to investigate and as far as I can tell it's working fine, closing.
